scripts/verify-jaeger-traces.sh

```diff
@@ -4,7 +4,17 @@ set -euo pipefail
 TRACE_DATA=$(curl -s "http://localhost:16686/api/traces?service=containerd&limit=0" \
     | jq '[ .data[].spans[].operationName ]')
 
-REQUIRED_OPS=("task_create" "task_wait" "task_start" "task_delete" "task_state" "wait" "shutdown" "shim_main_inner")
+REQUIRED_OPS=("task_create" "task_wait" "task_start" "task_delete" "task_state")
```

Comment:
```suggestion
PREFIX="containerd_shim_wasm::sandbox::shim::local"
REQUIRED_OPS=("$PREFIX::task_create" "$PREFIX::task_wait" "$PREFIX::task_start" "$PREFIX::task_delete" "$PREFIX::task_state")
```
Then you can combine it with wait and shutdown like before